### PR TITLE
Update remove-former-employee-step-5.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about onedrive URL

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8376

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee-step-5.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ You can also grant access to another user to access a former employee's OneDrive
 
 9. Add the user to **Site collection administrators** and select **OK**.
 
-10. The user will now be able to access the former employee's OneDrive using the OneDrive URL. 
+10. The user will now be able to access the former employee's OneDrive using the OneDrive URL. For more information, See [About OneDrive URLs](/onedrive/list-onedrive-urls#about-onedrive-urls)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-5.md]:
```suggestion
10. The user will now be able to access the former employee's OneDrive using the OneDrive URL. For more information, see [About OneDrive URLs](/onedrive/list-onedrive-urls#about-onedrive-urls).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. thanks

**Pull request merged**